app/controllers/users_controller.rb

```diff
@@ -8,6 +8,13 @@ def me
 
     # Store the parsed response in an instance variable
     @me = JSON.parse(get_me)
+
+    rescue RestClient::ExceptionWithResponse
+      if session[:oauth_response]
+        redirect_to users_home_path, danger: "Something Went Wrong. Please Check/Refresh your Access Token and Try Again"
```

Comment:
My comments regarding the messages in `login_controller.rb` also apply to this message and the others within this file (lines 16, 43, and 45).